sdk-commons/src/main/java/org/openmrs/maven/plugins/model/DistroProperties.java

```diff
@@ -8,6 +8,7 @@
 import org.openmrs.maven.plugins.utility.DistroHelper;
 
 import java.io.File;
+import java.io.FileNotFoundException;
```

Comment:
FileNotFoundException is a subclass of IOException, so catching IOException will also catch FileNotFoundException.